doc/getting-started/getting-started.rst

```diff
@@ -403,7 +382,7 @@ To generate a scenario configuration file and launch scripts:
 
    .. code-block:: bash
 
-      ./acrn_configurator.py
+      python3 acrn_configurator.py
```

Comment:
Don't need the ``python3`` now:
```suggestion
      ./acrn_configurator.py
```

Reply:
I tried to run ``./acrn_configurator.py`` command with v2.7 and it didn't work.

Reply:
> I tried to run `./acrn_configurator.py` command with v2.7 and it didn't work.

Well, I'll be hornswoggled.  Indeed the shebang was added to the python script after the 2.7 release so the ``python3`` is still needed until we checkout a newer version.